packages/web-components/src/components/footer/footer.scss

```diff
@@ -41,6 +41,21 @@
 :host(#{$dds-prefix}-footer),
 :host(#{$dds-prefix}-legal-nav) {
   display: block;
+  div.#{$prefix}--adjunct-links__container {
+    list-style: none;
+    padding: $spacing-09 0 $spacing-05;
+    padding-left: 1rem;
+    .#{$prefix}--adjunct-links__row {
+      @include carbon--make-row;
+      .#{$prefix}--adjunct-links__col {
+        padding-right: $spacing-05;
+        @include carbon--make-col(4, 12);
```

Comment:
This should probably be used in the breakpoint mixin to avoid a `max-width` in mobile viewports.

![image](https://user-images.githubusercontent.com/909118/123308402-9c55af00-d4f1-11eb-9931-ee5679635fe2.png)

Reply:
Can disregard, I saw your [comment](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/6271#pullrequestreview-690799697) about waiting for Raph's PR to get merged first.